### PR TITLE
[CTOR-554] Update the Prometheus kubernetes doc page

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: pp/integrations/plugin-packs/procedures/cloud-prometheus-kubernetes-api.md [hunk 1]
```diff
@@ -1,44 +1,475 @@
...
+
+</TabItem>
+</Tabs>
 
 ## Prerequisites
 
-### Centreon Plugin
+*Specify prerequisites that are relevant. You may want to just provide a link\n\
+to the manufacturer official documentation BUT you should try to be as complete\n\
+as possible here as it will save time to everybody.*
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-prometheus-kubernetes-api.md:94]: no prerequisites

File: pp/integrations/plugin-packs/procedures/cloud-prometheus-kubernetes-api.md [hunk 2]
```diff
@@ -1,44 +1,475 @@
...
+	--warning-allocated-pods='' \
+	--critical-allocated-pods='' \
+	--units='' \
+	--verbose
+```
+
+The expected command output is shown below:
+
+```bash
+OK: All nodes status are ok | '*nodes*#pods.allocated.count'=;;;;
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-prometheus-kubernetes-api.md:296]: @omercier is that sufficient?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-prometheus-kubernetes-api.md [hunk 3]
```diff
@@ -2,43 +2,469 @@
...
+	--warning-allocated-pods='' \
+	--critical-allocated-pods='' \
+	--units='' \
+	--verbose
+```
+
+La commande devrait retourner un message de sortie similaire à :
+
+```bash
+OK: All nodes status are ok | '*nodes*#pods.allocated.count'=;;;;
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-prometheus-kubernetes-api.md:291]:
```suggestion
OK: All nodes status are ok | '*nodes*#pods.allocated.count'=42;;;;
```

File: pp/integrations/plugin-packs/procedures/cloud-prometheus-kubernetes-api.md [hunk 4]
```diff
@@ -2,43 +2,472 @@
...
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/cloud-prometheus-kubernetes-api.md:294]:
```suggestion
OK: All nodes status are ok | '*nodes*#pods.allocated.count'=42;;;;
```

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**